gramdictru/wwwroot/css/site.css

```diff
@@ -440,4 +440,12 @@ a.contents-link {
 
 .page.is-searching .body-content {
     max-width: unset;
-}
\ No newline at end of file
+}
+
+:target {
+    border-top: 53px solid transparent;
+    margin-top: -53px;
```

Comment:
Nice! However, this looks like a magic number to me. It’s likely to break if e.g. someone changes the font size in .search-bar. Can we set .search-bar {height} and these two properties to the same [CSS variable](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_variables) search-bar-height?